clientconn.go

```diff
@@ -1438,6 +1443,28 @@ func (ac *addrConn) getReadyTransport() transport.ClientTransport {
 	return nil
 }
 
+// getTransport waits until the addrconn is ready or TF and returns the
+// transport or an error.
```

Comment:
I see the conditional on line 1456 returns an error on the addr conn being in state SHUTDOWN. Also, can you please reword the comment to something like "waits until the addrconn is ready and xyz happens as aresult (i.e. returning an error)...or the addr conn is tf then xyz2 happens.

Reply:
Done